core/modules/contact/contact.module

```diff
@@ -34,7 +34,7 @@ function contact_permission() {
       'title' => t('Use the site-wide contact form'),
     ),
     'access user contact forms' => array(
-      'title' => t("Use users' personal contact forms"),
+      'title' => t("Use personal contact forms"),
```

Comment:
Lets drop the double quotes now that there's not apostrophe (single quote) within the translatable string:
```suggestion
      'title' => t('Use personal contact forms'),
```